samples/nrf9160/nrf_cloud_agps/Kconfig

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "nRF Cloud and A-GPS Sample Settings"
+
+config CLOUD_BACKEND
```

Comment:
It's nrf_cloud sample, I don't think we have a choice here.

Reply:
Too much copy-pasting... Removed the option.